docs/usage/extract.rst

```diff
@@ -19,6 +19,9 @@ Examples
     # Extract the "src" directory but exclude object files
     $ borg extract my-files home/USERNAME/src --exclude '*.o'
 
+    # Extract only the object files
+    $ borg extract my-files 'sh:home/USERNAME/src/*.o'
+
     # Restore a raw device (must not be active/in use/mounted at that time)
```

Comment:
Good to add an example, but please use something more familiar to users (.pdf rather than .o). Not everybody is a developer.